packages/@lwc/rollup-plugin/src/__tests__/fixtures/expected_default_config_simple_app_css_resolver.js

```diff
@@ -1,104 +1,124 @@
-        key: 0
-      }, [api_dynamic($cmp.x)])];
-    }
-
-    var _tmpl = lwc.registerTemplate(tmpl);
-    tmpl.stylesheets = [];
-
-    if (_implicitStylesheets) {
-        tmpl.stylesheets.push.apply(tmpl.stylesheets, _implicitStylesheets);
-    }
-    tmpl.stylesheetTokens = {
-        hostAttribute: "x-foo_foo-host",
-        shadowAttribute: "x-foo_foo"
-    };
-
-    class Foo extends lwc.LightningElement {
-      constructor(...args) {
-        super(...args);
-        this.x = void 0;
-      }
-
-    }
-
-    lwc.registerDecorators(Foo, {
-      publicProps: {
-        x: {
-          config: 0
-        }
-      }
-    });
-
-    var _xFoo = lwc.registerComponent(Foo, {
-      tmpl: _tmpl
-    });
-
-    function tmpl$1($api, $cmp, $slotset, $ctx) {
-      const {
-        c: api_custom_element,
-        h: api_element
-      } = $api;
-      return [api_element("div", {
-        classMap: {
-          "container": true
-        },
-        key: 1
-      }, [api_custom_element("x-foo", _xFoo, {
-        props: {
-          "x": "1"
+(function (lwc, varResolver) {
+  "use strict";
+
+  function _interopDefaultLegacy(e) {
+    return e && typeof e === "object" && "default" in e ? e : { default: e };
+  }
+  
+  var varResolver__default = /*#__PURE__*/ _interopDefaultLegacy(varResolver);
+  
+  function stylesheet(hostSelector, shadowSelector, nativeShadow) {
+    return [
+      "\n",
+      nativeShadow
+        ? [
+            ":host {color: ",
+            varResolver__default["default"]("--lwc-my-color"),
+            ";}",
+          ].join("")
+        : [
+            hostSelector,
+            " {color: ",
+            varResolver__default["default"]("--lwc-my-color"),
+            ";}",
+          ].join(""),
+      "\n",
+    ].join("");
+  }
+  var _implicitStylesheets = [stylesheet];
+  
+  function tmpl($api, $cmp, $slotset, $ctx) {
+    const { d: api_dynamic, h: api_element } = $api;
+    return [
+      api_element(
+        "div",
+        {
+          key: 0,
         },
-        key: 0
-      }, [])])];
+        [api_dynamic($cmp.x)]
+      ),
+    ];
+  }
+
+  var _tmpl = lwc.registerTemplate(tmpl);
+  tmpl.stylesheets = [];
+
+  if (_implicitStylesheets) {
+    tmpl.stylesheets.push.apply(tmpl.stylesheets, _implicitStylesheets);
+  }
+  tmpl.stylesheetTokens = {
+    hostAttribute: "x-foo_foo-host",
+    shadowAttribute: "x-foo_foo",
+  };
+
+  class Foo extends lwc.LightningElement {
+    constructor(...args) {
+      super(...args);
+      this.x = void 0;
     }
-
-    var _tmpl$1 = lwc.registerTemplate(tmpl$1);
-    tmpl$1.stylesheets = [];
-    tmpl$1.stylesheetTokens = {
-        hostAttribute: "x-app_app-host",
-        shadowAttribute: "x-app_app"
-    };
-
-    class App extends lwc.LightningElement {
-      constructor() {
-        super();
-        this.list = [];
-      }
-
+  }
+
+  lwc.registerDecorators(Foo, {
+    publicProps: {
+      x: {
+        config: 0,
+      },
+    },
```

Comment:
Wonder if this would set off any eslint/prettier warnings for customer components in their local dev setup.

Reply:
this is the compiled code, it shouldn't be linted right?

Reply:
It should not be. It's probably unlikely.